style.css

```diff
@@ -1,27 +1,23 @@
 #navbar {
-    position: fixed;
-    height: 100%;
-    min - weight : 300px;
-    width : 300px;
```

Comment:
- [ ] Kindly organize css spaces.

Reply:
- [ ] Good job adding an inline comment as requested by the previous reviewers :clap: :clap:  Kindly add the reason of your of suggestion as they recommended too. Instead of "Kindly organize CSS spaces." only I suggest adding "Kindly organize CSS spaces because it would make your code look more professional and cleaner"